include/GloryDuckTables2019Lkl.h

```diff
@@ -0,0 +1,74 @@
+//////////////////////////////////////////////////////////////////////
+// Binned flux likelihood
+//////////////////////////////////////////////////////////////////////
+
+#ifndef GLORYDUCKLKL
+#define GLORYDUCKLKL
+
+#include "TObjArray.h"
+#include "TCanvas.h"
+
+#include "Lkl.h"
+
+class GloryDuckTables2019Lkl : public Lkl
+{
+ public:
+  // constructors
+  GloryDuckTables2019Lkl(TString inputString="");
+  
+  // destructor
+  virtual ~GloryDuckTables2019Lkl();
+ 
+  // make the TObjArray own its contents
+  void SetOwner(Bool_t set=kTRUE)            {fSampleArray->SetOwner(set);}
```

Comment:
Why is this needed? it is not in the Fermi version.
In any case either setting ownership is needed or not, but should not be a decision of the user, should it?
I mean, if it is needed, just hard-code the call in the appropriate place, right?

Reply:
It is not needed (at least I don't use it). I added it when I created the TObjectArray for the class (inspired by the JointLkl class) and at that time I copied all available function concerning the TObjectArray just in case I needed them later... I will delete that line.